kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/ints/IntMatchers.kt

```diff
@@ -7,9 +7,11 @@ import io.kotest.matchers.shouldNot
 
 fun beBetween(a: Int, b: Int) = between(a, b)
 fun between(a: Int, b: Int): Matcher<Int> = object : Matcher<Int> {
-  override fun test(value: Int) = MatcherResult(value in a..b,
-    "$value is not between ($a, $b)",
-    "$value is between ($a, $b)")
+  override fun test(value: Int) = MatcherResult(
+     value in a..b,
+    "$value should be between ($a, $b)",
```

Comment:
Should we mention explicitly that check is inclusive of both a and b like we have mentioned for byte matchers.
https://github.com/kotest/kotest/blob/c1b1b1256b0894e01f065fe1ead4e73d73ed6ba6/kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/bytes/byte.kt#L8-L12

Reply:
Done.